### ISSUE TITLE
spec: Package name

### ISSUE THREAD

**Issue by akutz** [container-storage-interface/spec#51]
Currently the protobuf has a package name of _spec_. I propose this be renamed to _csi_ as it's much more reflective of the actual specification. The package name could also follow the Sun naming conventions, but I think a simple package name of _csi_ will suffice.

The package name is important as it dictates the namespace for all of the protobuf messages and gRPC services. Right now a `ListVolumesRequest` gets generated as `spec.ListVolumesRequest`. That's not good. As a package name `spec` presents too much of a potential namespace collision. This is especially true with Go's implementation of protobuf and gRPC as both register message and service types with a central registry. Currently all of the CSI types would be in the package (namespace) of `spec`. 

The first three lines of the CSI protobuf file with the proposed change might resemble the following:

```protobuf
syntax = "proto3";
package csi;
option go_package = "csi";
```

tl;dr - Change the CSI protobuf's **package** from `spec` to `csi`.

**Comment by jdef** [comment]
Sounds good to me.

**Comment by akutz** [comment]
Hi @jdef,

I'm sakutz at gmail and ssakutz on Twitter. I have been trying to figure out your contact information, but for the life of me I am unable to do so. I'd like to chat with you if/when you're available. Please give me a shout if/when you have some time. Thanks!

-- 
-a

p.s. I'll delete this comment once I know you've seen it.

**Comment by jdef** [comment]
the generated golang protos are published to `package csi`. closing this out. please re-open if needed
